src/gwt/src/org/rstudio/studio/client/projects/ui/prefs/ProjectEditingPreferencesPane.java

```diff
@@ -58,6 +60,13 @@ public ProjectEditingPreferencesPane(final SourceServerOperations server)
       numSpacesForTab_.setWidth("36px");
       add(numSpacesForTab_);
 
+      LayoutGrid useNativePipeLabeled = new LayoutGrid(1, 2);
```

Comment:
Can we tweak the text a bit?

- Use italics
- Add a bit of extra padding at the bottom
- Include a period at the end of the sentence

Reply:
@kevinushey Do you mean like just put "(requires R 4.1+)" in italics?

Because I can do that for the global preferences which uses a Checkbox

<img width="338" alt="image" src="https://user-images.githubusercontent.com/7817881/164081069-65b04792-0396-4051-aa9f-424565f21fff.png">

but I don't know how to do that for the Project Preferences, as I don't think the FormLabel widget has an `asHtml` param or anything like that
<img width="403" alt="image" src="https://user-images.githubusercontent.com/7817881/164081199-08cc5851-bafe-4aee-ab30-78abdbc91553.png">

If you meant to set the whole line in italics, I could do that in the CSS, but that seems strange

Reply:
Sorry, I was referring to the "Use (default) to inherit the global default setting" text; I don't think we need to tweak the other text.

Reply:
Ah, that makes more sense and you're right -- we already have a CSS style for that bit, I just forgot to apply it

Reply:
Better?

<img width="546" alt="image" src="https://user-images.githubusercontent.com/7817881/164302012-7b0b9b4b-6942-4cd4-b5e0-dfc958bb9a13.png">